src/garage/torch/algos/vpg.py

```diff
@@ -208,7 +206,7 @@ def _compute_loss(self, itr, paths, valids, obs, actions, rewards):
         valid_objectives = filter_valids(objective, valids)
         return -torch.cat(valid_objectives).mean()
 
-    def _compute_kl_constraint(self, obs):
+    def compute_kl_constraint(self, obs):
```

Comment:
this name is not accurate, because not all policy gradient algorithms have a KL constraint.